src/main/java/uk/gov/digital/ho/hocs/info/utils/EmailDomainValidator.java

```diff
@@ -12,13 +13,17 @@
 public class EmailDomainValidator implements ConstraintValidator<EmailDomainWhitelisted, String> {
 
     @Value("${user.email.whitelist}")
-    private List<String> whitelistedDomains = asList("homeoffice.gov.uk");
+    private List<String> whitelistedDomains = Collections.emptyList();;
 
     @Override
     public void initialize(EmailDomainWhitelisted emailDomainWhitelisted) { }
 
     @Override
     public boolean isValid(String emailAddress, ConstraintValidatorContext cxt) {
-      return emailAddress == null || emailAddress.isBlank() || whitelistedDomains.stream().anyMatch(emailAddress::endsWith);
+        return emailAddress != null
+                && !emailAddress.isBlank()
+                && whitelistedDomains
+                    .stream()
+                    .anyMatch(domain -> emailAddress.toLowerCase().endsWith(String.format("@%s", domain)));
```

Comment:
While `foo@example.com` and `foo@EXAMPLE.COM` are the same email address, according to RFC 5322  `foo@example.com` and `FOO@example.com` aren't -- although almost all mailservers these days will treat them as such.

If there's a handy `normalizeEmail` method handy that could be better than a simple`.toLowerCase()`.

Reply:
toLower() is sufficient in this situation.